app/assets/javascripts/invitations.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(function() {
+    $('form#new_invitation').submit(function() {
+        //console.log('TEST');
+        var valuesToSubmit = $(this).serialize();
+        $.ajax({
+            type: "POST",
+            url: ('invitations'),
+            data: valuesToSubmit,
+            dataType: "JSON"
+        }).success(function (json) {
+            //console.log("success", json);
+            var messageTextArea = document.getElementById('message');
```

Comment:
If you use jQuery already then please use it's features -  here we could find element with ```$('#message')```
This is more cross-browser.